open_discussions/urls.py

```diff
@@ -32,6 +32,7 @@
     url(r"", include("notifications.urls")),
     url(r"", include("embedly.urls")),
     url(r"", include("search.urls")),
+    url(r"", include("open_widget_framework.urls")),
```

Comment:
There's a feature flag for the UI but not for the backend. I haven't looked at the backend code, do you think it's ready for release, or at least will deny non-admin users access appropriately?

Reply:
Looks like the urls to modify widgets is protected but not the one to modify lists is not. I think I'd like to actually fix this in an upstream PR.